server/aws-lsp-codewhisperer/src/language-server/telemetryService.ts

```diff
@@ -209,6 +225,18 @@ export class TelemetryService extends CodeWhispererServiceToken {
         modificationPercentage: number
         customizationArn?: string
     }) {
+        if (this.enableTelemetryEventsToDestination) {
```

Comment:
I suggest to set `enableTelemetryEventsToDestination` to `false` as default value explicitly where it is first defined first https://github.com/aws/language-servers/pull/574/files#diff-9651c5ce85ab6404bc20ff59b61b93254f540d5e677f455b75e3d17ce5c07f08R36

Reply:
I'll set as default false in the constructor itself.